### ISSUE TITLE
BAD SEMANTICS

### ISSUE THREAD

**Issue by JasonWarrenUK** [FAC29A/hot-gitato#8]
Ooops, I did some bad things, such as...

- Using generic divs
- Duplicating some ids

PLS CAN SUM1 FIX, I AM A SIMPL BOI HU NOT NO HAO 2 CODE

**Comment by Frannerz** [comment]
Jason- have you created a branch/made edits/staged edits/created a pull request?

**Comment by Luciensday** [comment]
maybe merge my `add-styling` branch firstly and then create a branch to fix these issues.

**Comment by JasonWarrenUK** [comment]
@Frannerz, no, this issue relates to problems I introduced while I fixed your typos

So they are all in the `main` branch, and I want someone to fix it :)

@Luciensday, I've deliberately done this before reviewing and merging your branch, because I want to practice having lots of branches and versions happening at the same time.
